scripts/train.py

```diff
@@ -248,7 +248,7 @@ def dummy_init_fn(module: torch.nn.Module) -> None:
                     )
                     cfg.save_interval_unsharded = cfg.save_interval
 
-                if cfg.save_num_unsharded_checkpoints_to_keep < 1:
+                if cfg.save_num_unsharded_checkpoints_to_keep == 0:
                     log.warning(
```

Comment:
What if `save_num_checkpoints_to_keep` is also 0?

Reply:
it then assumes that you did not want to keep checkpoints at all!

Reply:
-1 assumes you want to save all checkpoints and so I made it ``==0`` instead of ``< 1``.